packages/postcss-reduce-initial/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "acquire": "node ./src/script/acquire.mjs",
     "prebuild": "rimraf dist",
-    "build": "cross-env BABEL_ENV=publish babel src --config-file ../../babel.config.json --out-dir dist --ignore '**/__tests__/,src/script'",
+    "build": "babel src --config-file ../../babel.config.json --out-dir dist --ignore \"**/__tests__/\"",
```

Comment:
I've moved the `script` directory out of `src`  for `postcss-reduce-initial` to make it harder to publish the `script` directory by mistake.
The `build` script for `postcss-reduce-initial` had a more complex ignore pattern, because it was the only package with the `scripts` directory  inside the `src` directory. On Windows, removing cross-env probably changed the way some globs were interpreted, so the `publish` script placed a copy of the `script` directory inside `dist`. Jest tried to run the tests inside `dist/scripts/__tests__/ `, but the imports then did not work, causing the build to fail.